feature-swap-core/src/main/java/io/novafoundation/nova/feature_swap_core/data/assetExchange/conversion/types/hydra/RealHydraDxAssetConversion.kt

```diff
@@ -50,6 +50,10 @@ class RealHydraDxAssetConversion(
 
     private val conversionSources: List<HydraDxConversionSource> = createSources()
 
+    override suspend fun sync() {
+        conversionSources.forEach { it.sync() }
```

Comment:
Lets make it async so some slow source wont stop everything else from syncing
We have mapAsync helper for that